packages/eas-cli/src/commands/build/create.ts

```diff
@@ -75,3 +86,51 @@ export default class BuildCreate extends Command {
     await buildAsync(commandCtx);
   }
 }
+
+async function promptForPlatformAsync(): Promise<RequestedPlatform> {
+  const { platform } = await promptAsync({
+    type: 'select',
+    message: 'Which platforms would you like to build for?',
+    name: 'platform',
+    choices: [
+      {
+        title: 'All',
+        value: RequestedPlatform.All,
+      },
+      {
+        title: 'iOS',
+        value: RequestedPlatform.iOS,
+      },
+      {
+        title: 'Android',
+        value: RequestedPlatform.Android,
+      },
+    ],
+  });
+  return platform;
+}
+
+async function ensureProjectConfiguredAsync(projectDir: string): Promise<void> {
+  if (!(await fs.pathExists(path.join(projectDir, 'eas.json')))) {
+    log.warn('This project is not confgured for EAS Build - eas.json is missing');
+    const confirm = await confirmAsync({
+      message: 'Do you want to configure this project to be compatible with EAS Build?',
+    });
+    if (confirm) {
+      await configureAsync({
+        projectDir,
+        platform: RequestedPlatform.All,
```

Comment:
Shouldn't this be set to the requested platform?

Reply:
that is the main reason I'm running this before the platform is resolved.

I think that platform-specific configure make sense only to update the project not for initial configuration, especially since we removed credentials generation